code/standards/checkstyle.xml

```diff
@@ -13,13 +14,16 @@
       <property name="allowUndeclaredRTE" value="true"/>
       <property name="allowMissingParamTags" value="true"/>
       <property name="allowMissingReturnTag" value="true"/>
+      <property name="allowMissingThrowsTags" value="true" />
       <property name="allowMissingPropertyJavadoc" value="true"/>
     </module>
     <module name="JavadocType"/>
     <module name="JavadocVariable">
       <property name="scope" value="public"/>
     </module>
-    <module name="ConstantName"/>
+    <module name="ConstantName">
+      <property name="applyToPrivate" value="false" />
```

Comment:
Okay, so should we put this with a comment that says "off for now to focus on stuff later, but apply later"?